src/SolverInterface/lpqp_to_conic.jl

```diff
@@ -154,9 +154,11 @@ function loadproblem!(m::LPQPtoConicBridge, c, A, b, constr_cones, var_cones)
         # rsoc has x'x <= 2pq
         # quadratic form is x'x <= pq
         diagvec = ones(length(rsocconstr_idx))
-        diagvec[rsoc_start_idx] = 1/sqrt(2)
+        for rsoc_idx in rsoc_start_idx
+            diagvec[rsoc_idx] = 1/sqrt(2)
+        end
         Alin = [ Alin spzeros(size(Alin,1),length(rsocconstr_idx))
-        [m.Arsoc spzeros(size(m.Arsoc,1),length(socconstr_idx))] spdiagm(diagvec) ]
+        [m.Arsoc spzeros(size(m.Arsoc,1),length(socconstr_idx))] @compat spdiagm(0 => diagvec) ]
```

Comment:
I don't think the compat macro is needed here. `SparseArrays.spdiagm(0 => diagvec)` should do the trick.

Reply:
I tried this syntax in 0.6, it returns `MethodError: no method matching spdiagm(::Pair{Int64,Array{Int64,1}})`

Reply:
Sorry, I will fix it like you suggested in https://github.com/JuliaOpt/GLPKMathProgInterface.jl/pull/52

Reply:
```julia
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: https://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.6.3 (2018-05-28 20:20 UTC)
 _/ |\__'_|_|_|\__'_|  |  Official http://julialang.org/ release
|__/                   |  x86_64-pc-linux-gnu

julia> import Compat.SparseArrays

julia> SparseArrays.spdiagm(0 => [1, 2])
2×2 SparseMatrixCSC{Int64,Int64} with 2 stored entries:
  [1, 1]  =  1
  [2, 2]  =  2
```